ister.py

```diff
@@ -1410,6 +1410,33 @@ def download_ister_conf(uri, timeout=15):
                                 .format(uri, err))
 
 
+def enable_root_ssh_login():
+    """Configure current environment to passwordless root SSH log in.
+    NOTE: this is not about configuring the final system, this is about
+    configuring the currently running installer system.
+
+    The idea here is to open a possibility for the user to reconfigure the
+    installer system/environment or follow the installation log during the
+    installation process from an outside script.
+   """
+
+    LOG.debug("Enable paswordless root SSH login")
+    # Set empty root password.
+    run_command("usermod -p '' root")
+    # Create a minimal SSH server config file allowing for root login and
+    # enables SFTP.
+    sshdir = os.path.join("etc", "ssh")
+    os.makedirs(sshdir, mode=0o0755, exist_ok=True)
+    with open(os.path.join(sshdir, "sshd_config"), "a") as fobj:
+        os.fchmod(fobj.fileno(), 0o600)
+        fobj.write("# Added by ister due to 'ister.root_ssh'.\n"
+                   "PermitRootLogin yes\n"
+                   "PermitEmptyPasswords yes\n"
+                   "Subsystem sftp /usr/libexec/sftp-server\n")
+    # Restart the SSH service.
+    run_command("systemctl restart sshd")
```

Comment:
Mind me poking this over to use sshd.socket rather than the service (I have a few conflicts I am resolving so I can do this when I push those)? Since we generally recommend using the socket and so using the service here might conflict with somebody else who built their own installation own image.

Reply:
Not 100% sure I understood the request correctly, but I'll change this to 'systemctl restart sshd.socket'.